azure-specialized-workloads/oracle/recommendations.yaml

```diff
@@ -0,0 +1,201 @@
+- description: Implement a regional replication strategy for Oracle to meet your workload requirements
+  aprlGuid: dfeb9c7a-7dae-4751-9625-b23a7160a3e1
+  recommendationTypeId: null
+  recommendationControl: Business Continuity
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/oracle
+  recommendationMetadataState: Active
+  longDescription:
+    Regional replication is a key strategy to ensure business continuity and disaster recovery for your Oracle workloads. Implement a regional replication strategy to meet your workload requirements.
+  potentialBenefits: Ensure business continuity in case of regional failure.
+  pgVerified: false
+  publishedToLearn: false
+  automationAvailable: no
+  tags: null
+  learnMoreLink:
+    - name: Business continuity and disaster recovery considerations for Oracle Database@Azure
+      url: "https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/scenarios/oracle-iaas/oracle-disaster-recovery-oracle-database-azure"
+    - name: Learn about Oracle Maximum Availability Architecture for Oracle Database@Azure
+      url: "https://docs.oracle.com/en/solutions/oracle-maa-db-at-azure/#GUID-7723E2B1-9588-40BC-88BE-44637B1AF0D9"
+    - name: Oracle Database@Azure Evaluations by Oracle MAA
+      url: "https://docs.oracle.com/en/database/oracle/oracle-database/21/haovw/db-azure1.html#GUID-91572193-DF8E-4D7A-AF65-7A803B89E840"
+
+
+- description: Implement a strategy for resiliency that is tailored to your Oracle workload requirements
```

Comment:
This appears to be more architectural guidance than a configuration for the Oracle resource that can be configured to improve the resiliency of the Oracle resource deployment. As such, wouldn't this better be addressed through WAF guidance?

Reply:
The recommendation is agreed by relevant teams and as such we would like to keep it

Reply:
Agree with @ehaslett this reads as a generic best practice rather than an actionable reliability recommendation. We also already have WAF recommendations RE:05 that cover the same topic.

Reply:
@basimolimajeed , due to the APRL requirements @ehaslett mentions [above](https://github.com/azure/azure-proactive-resiliency-library-v2/pull/463#discussion_r1819500407), this should be moved to WAF.